styles.scss

```diff
@@ -115,6 +115,30 @@ body {
       }
     }
 
+    .clicked-wrong {
+      background-color: darken($darker-red, 10%);
+      &:hover {
+        cursor: default;
+        background-color: darken($darker-red, 10%);
+      }
+      p {
+        color: $whiteish;
+        text-shadow: -2px 0 $blackish, 0 2px $blackish, 2px 0 $blackish, 0 -2px $blackish;
+      }
+    }
+
+    .clicked-correct {
+      background-color: lighten($lighter-red, 10%);
+      &:hover {
+        cursor: default;
+        background-color: lighten($lighter-red, 10%);
+      }
+      p {
```

Comment:
the same. In general, code added in this PR into scss file should be nicely formatted. Now looks like shit.